source/crud/tests/unified/replaceOne-hint.yml

```diff
@@ -64,10 +64,8 @@ tests:
                     q: *filter
                     u: *replacement
                     hint: _id_
-                    multi:
-                      $$unsetOrMatches: false
-                    upsert:
-                      $$unsetOrMatches: false
+                    multi: { $$unsetOrMatches: false }
+                    upsert: { $$unsetOrMatches: false }
```

Comment:
There are no corresponding JSON changes here, since I was just changing the YAML formatting. This will make it easier to search and remove all of these instances when [DRIVERS-2271](https://jira.mongodb.org/browse/DRIVERS-2271) is implemented.